src/common/types.ts

```diff
@@ -167,6 +167,10 @@ export interface IRowRendererProps<TRow> {
   onRowExpandToggle?(event: RowExpandToggleEvent): void;
 }
 
+export interface CustomRowRendererProps<TRow> extends IRowRendererProps<TRow> {
+  renderBaseRow: React.FunctionComponent<IRowRendererProps<TRow>>;
+}
+
```

Comment:
We should only export the public interfaces instead of `export * from './common/types';`. We can check which ones we need in a separate PR